src/earn/EarnEnterAmount.tsx

```diff
@@ -362,25 +408,21 @@ function EarnEnterAmount({ route }: Props) {
             testID="EarnEnterAmount/PrepareTransactionError"
           />
         )}
-        <EarnProceed
-          tokenAmount={tokenAmount}
-          localAmount={localAmount}
-          token={token}
-          amountEnteredIn={enteredIn}
-          onPressProceed={onPressContinue}
-          onPressInfo={onPressInfo}
+        <Button
+          onPress={() =>
+            tokenAmount &&
+            onPressContinue({ tokenAmount, localAmount, token, amountEnteredIn: enteredIn })
+          }
+          text={t('earnFlow.enterAmount.continue')}
+          size={BtnSizes.FULL}
           disabled={disabled}
-          loading={isPreparingTransactions}
-          pool={pool}
+          style={styles.continueButton}
+          showLoading={isPreparingTransactions}
+          testID="EarnEnterAmount/Continue"
         />
         <KeyboardSpacer />
       </KeyboardAwareScrollView>
       <InfoBottomSheet infoBottomSheetRef={infoBottomSheetRef} />
-      <EarnAddCryptoBottomSheet
```

Comment:
can we remove this component now?

Reply:
i'm surprised knip didn't flag this

Reply:
I guess it didn't because tests where using it